internal/connector/internal/api/admin/private/api_connector_clusters_admin.go

```diff
@@ -743,6 +743,102 @@ func (a *ConnectorClustersAdminApiService) GetConnector(ctx _context.Context, co
 	return localVarReturnValue, localVarHTTPResponse, nil
 }
 
+/*
+GetConnectorCluster Get a connector cluster
+ * @param ctx _context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+ * @param connectorClusterId The id of the connector cluster
+@return ConnectorCluster
+*/
+func (a *ConnectorClustersAdminApiService) GetConnectorCluster(ctx _context.Context, connectorClusterId string) (ConnectorCluster, *_nethttp.Response, error) {
```

Comment:
not quite sure where this came from. Connectors related change that appeared after `make generate`.
ping @lburgazzoli